bindings/libdnf5/rpm.i

```diff
@@ -61,6 +61,7 @@
 %template(VectorNevra) std::vector<libdnf5::rpm::Nevra>;
 %template(VectorNevraForm) std::vector<libdnf5::rpm::Nevra::Form>;
 %template(PairBoolNevra) std::pair<bool, libdnf5::rpm::Nevra>;
+%template(cmp_nevra) libdnf5::rpm::cmp_nevra<libdnf5::rpm::Nevra>;
```

Comment:
I think the Swig template identifier should held the specialization type as it needs to be unique, e.g. <code>%template(CmpNevraNevra) libdnf5::rpm::cmp_nevra<libdnf5::rpm::Nevra, libdnf5::rpm::Nevra></code>. But please ask @jrohel for the review. I have no experience with Swig.

Reply:
I'm thinking. `cmp_nevra` is a function template. SWIG supports function overloading. So it shouldn't be a problem in the future to define multiple functions with the same name (multiple specializations of a function template under the same name and with different types of arguments).
For example:
```
%template(cmp_nevra) libdnf5::rpm::cmp_nevra<libdnf5::rpm::Nevra, libdnf5::rpm::Nevra>
%template(cmp_nevra) libdnf5::rpm::cmp_nevra<libdnf5::rpm::Package, libdnf5::rpm::Package>
```
The problem might be the target language. SWIG supports multiple languages. Libdnf5 has bindings for Python, Ruby and Perl. So I looked at support for these 3 languages. It looks like overloaded functions are supported for all 3 languages. So we don't need to have a unique name in this case.